docs/testing-be.md

```diff
@@ -42,13 +74,10 @@ but is not used by defalt when running tests locally.
 ## Source code linting
```

Comment:
Should this be turned into a subsection of "Running tests"?

Reply:
Yeah, it probably can be at this point.